pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -47,28 +47,39 @@ import (
 	"github.com/tidwall/gjson"
 )
 
-// Image mimics a `docker image inspect` object.
-// From https://github.com/moby/moby/blob/v20.10.1/api/types/types.go#L340-L374
+// From https://github.com/moby/moby/blob/v26.1.2/api/types/types.go#L34-L140
 type Image struct {
```

Comment:
Updated with latest definition and reordered a but for readability.
- moved down deprecated fields
- added VirtualSize, Parent, Variant